File: Data Structures & Algorithms/monotonic-array/submission-1.cpp
class Solution
{
public:
    bool isMonotonic(vector<int> &nums)
    {
        int t = 0;
        for (int i = 1; i < nums.size(); i++)
        {
            auto lastT = t;
            auto a = nums[i] - nums[i - 1];
            if(a!= 0) t = nums[i] - nums[i - 1];
            if (t != 0 && lastT * t < 0) return false;
        }

        return true;
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.cs
public class Solution {
    public int[] TopKFrequent(int[] nums, int k) {
        Dictionary<int,int> dict = new Dictionary<int, int>();
        foreach (var item in nums)
        {
            dict[item] = dict.GetValueOrDefault(item, 0)+1;
        }

        var a = dict.OrderByDescending(x=>x.Value).ToDictionary();
        var b = a.Keys.Take(k).ToArray();
        return b;
    }
}
